include/cpptrace/basic.hpp

```diff
@@ -43,6 +43,14 @@
  #define CPPTRACE_FORCE_NO_INLINE __attribute__((noinline))
 #endif
 
+#ifdef _MSC_VER
+ #define CPPTRACE_FORCE_INLINE __forceinline
+#elif defined(__clang__) || defined(__GNUC__)
+ #define CPPTRACE_FORCE_INLINE __attribute__((always_inline)) inline
+#else
+ #define CPPTRACE_FORCE_INLINE inline
```

Comment:
I'd prefer `inline` to be treated as a separate concern from force inline. `inline` is primarily an ODR thing (despite having some effect on inlining heuristics) while these attributes I expect the optimizer to treat as a requirement.